plugins/Kaleidoscope-DynamicTapDance/src/kaleidoscope/plugin/DynamicTapDance.cpp

```diff
@@ -41,7 +41,7 @@ void DynamicTapDance::updateDynamicTapDanceCache() {
   map_[0] = 0;
 
   while (pos < storage_base_ + storage_size_) {
-    uint16_t raw_key = Kaleidoscope.storage().read(pos);
+    uint16_t raw_key = Runtime.storage().read(pos);
```

Comment:
I'm ok with this here, especially since we were mixing the two, but want to be careful about Kaleidoscope->Runtime changes, since we encourage user code to use the 'Kaleidoscope' name

Reply:
That's what I thought: User code in sketches uses `::Kaleidoscope`, but internal (and plugin?) code uses `::kaleidoscope::Runtime`.  If you'd rather have plugin code also use the `::Kaleidoscope`, that's okay with me, but I'd prefer to use `Runtime` in plugins, since that's the name that matches the file that's being included.  Does that make sense?

Reply:
I think that us standardizing on Runtime in our plugins is right.